zeppelin-web/src/app/interpreter/interpreter.controller.js

```diff
@@ -737,6 +741,24 @@ function InterpreterCtrl($rootScope, $scope, $http, baseUrlSrv, ngToast, $timeou
     }
   }
 
+  $scope.shouldEdit = function (form, settingId) {
```

Comment:
I would like to highlight that the permissions check is only done when clicking "Save" after "Edit". 
Although not perfect, this allows the user to check what permissions are set. Suggestions are welcome.